tests/service/CSharpProjectAnalysis.fs

```
@@ -0,0 +1,90 @@
+            |> ignore
+        failwith (builder.ToString())
+
+    results.ProjectContext.GetReferencedAssemblies()
+        |> List.map(fun x -> x.SimpleName, x)
+        |> dict
+
+[<Test>]
+let ``Test that csharp references are recognized as such`` () = 
+    let csharpAssembly = typeof<CSharpClass>.Assembly.Location
+    let table = getProjectReferences([csharpAssembly], None, None)
+    let ass = table.["CSharp_Analysis"]
+    match ass.Contents.Entities |> Seq.tryFind (fun e -> e.DisplayName = "CSharpClass") with
+    | Some found ->
+        // this is no F# thing
+        found.IsFSharp |> shouldEqual false
+        
+        // Check that we have members
+        let members = found.MembersFunctionsAndValues |> Seq.map (fun e -> e.CompiledName, e) |> dict
+        members.ContainsKey ".ctor" |> shouldEqual true
+        members.ContainsKey "Method" |> shouldEqual true
+        members.ContainsKey "Property" |> shouldEqual true
+        members.ContainsKey "Event" |> shouldEqual true
+        members.ContainsKey "InterfaceMethod" |> shouldEqual true
+        members.ContainsKey "InterfaceProperty" |> shouldEqual true
+        members.ContainsKey "InterfaceEvent" |> shouldEqual true
+
+        //// Check that we get xml docs
+        //String.IsNullOrWhiteSpace(members.[".ctor"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["Method"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["Property"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["Event"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["InterfaceMethod"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["InterfaceProperty"].XmlDocSig) |> shouldEqual false
+        //String.IsNullOrWhiteSpace(members.["InterfaceEvent"].XmlDocSig) |> shouldEqual false
+
+        ()
+    | None -> ()
```

I suppose that the code should fail in this case? (If the entity was not found, then something else went wrong...)